packages/config/config/devices/0x045a/Z-CM-V01.json

```diff
@@ -0,0 +1,218 @@
+{
+	"manufacturer": "ZVIDAR",
+	"manufacturerId": "0x045a",
+	"label": "Z-CM-V01",
+	"description": "Smart Curtain Motor",
+	"devices": [
+		{
+			"productType": "0x0904",
+			"productId": "0x0507"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Set To Start Holding Hands",
+			"description": "This parameter can be used to set the motor open hand start function",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
```

Comment:
```suggestion
```
Unnecessary if there are `options` and `allowManualEntry` is `false`. Please remove these for all other parameters where this condition is true.